test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -47,4 +47,44 @@ else
     exit 1
 fi
 
+# create asset with no manager, no freezer, and no clawback
+${gcmd} asset create --creator "${ACCOUNT}" --no-manager --no-freezer --no-clawback --name "${ASSET_NAME}" --unitname iamisc --total 1000000000000 --asseturl "${ASSET_URL}"
+
+IMMUTABLE_ASSET_ID=$(${gcmd} asset info --creator $ACCOUNT --unitname iamisc|grep 'Asset ID'|awk '{ print $3 }')
+
+MANAGER_ADDRESS=$(${gcmd} asset info --assetid ${IMMUTABLE_ASSET_ID} |grep 'Manager address'|awk '{ print $3 }')
+RESERVE_ADDRESS=$(${gcmd} asset info --assetid ${IMMUTABLE_ASSET_ID} |grep 'Reserve address'|awk '{ print $3 }')
+FREEZE_ADDRESS=$(${gcmd} asset info --assetid ${IMMUTABLE_ASSET_ID} |grep 'Freeze address'|awk '{ print $3 }')
+CLAWBACK_ADDRESS=$(${gcmd} asset info --assetid ${IMMUTABLE_ASSET_ID} |grep 'Clawback address'|awk '{ print $3 }')
+
+if [ "$MANAGER_ADDRESS" = "" ] \
+    && [ "$RESERVE_ADDRESS" = "$ACCOUNT" ] \
+    && [ "$FREEZE_ADDRESS" = "" ] \
+    && [ "$CLAWBACK_ADDRESS" = "" ]; then
+    echo ok
+else
+    date '+asset-misc immutable asset info error %Y%m%d_%H%M%S'
+    exit 1
+fi
+
+# create asset with a manager that is different from the creator
+${gcmd} asset create --creator "${ACCOUNT}" --manager "${ACCOUNTB}" --name "${ASSET_NAME}" --unitname dma --total 1000000000000 --asseturl "${ASSET_URL}"
```

Comment:
I'd prefer that this sets all of the different address types, not just manager, and that it use a different address for each.  That way we are testing against simple cut and paste bugs where, for example, using `--freezer X` actually sets the reserve address.